maas-schemas/schemas/core/components/common.json

```diff
@@ -50,6 +50,19 @@
       "maxLength": 64,
       "examples": ["joe.customer@example.com"]
     },
+    "customerReference": {
```

Comment:
Does this have any usage or only the ts version is?

Reply:
Good question. I imagine they would be used in the backend admin API once it is implemented but perhaps we should not hurry with merging them unless we have similar needs elsewhere.

Reply:
Also, there is this entire discussion about which schemas should be internal and which should be public. Perhaps, I'll close this now and open it again later.